cfgov/jinja2/v1/index.html

```diff
@@ -5,8 +5,8 @@
 {%- endblock %}
 
 {% block desc -%}
-    Our vision is a consumer finance marketplace that works for American consumers,
-    responsible providers, and the economy as a whole.
+    {{ 'Our vision is a consumer finance marketplace that works for American consumers, ' +
+       'responsible providers, and the economy as a whole.' }}
```

Comment:
I wasn't 100% sure if the multiple spaces before `responsible` would be shown literally in e.g. Google search results... but thinking about it further HTML strips out multiple spaces anyway, so it should be fine. Reverted in https://github.com/cfpb/cfgov-refresh/pull/1990/commits/d84bb713d82e95825c48eb92485f52cc24f10f9b